learn_script.py

```diff
@@ -4,39 +4,36 @@
 import shlex
 import subprocess
 import sys
-
+import argparse
 
 try:
     from ansimarkup import parse, ansiprint
 except:
-    ansiprint = lambda x: print(re.sub('<.*?>', '', x))
-
-
-script = open(sys.argv[1]).readlines()
+    def ansiprint(x): return print(re.sub('<.*?>', '', x))
 
-help_txt = []
```

Comment:
did you test this remove? this seems to cause an error on line 107

Reply:
I tested it, and everything went well here :thinking: